lib/main_common.pm

```diff
@@ -225,7 +225,7 @@ sub kdestep_is_applicable {
 }
 
 sub consolestep_is_applicable {
-    return !get_var("INSTALLONLY") && !get_var("DUALBOOT") && !get_var("RESCUECD");
+    return !get_var("INSTALLONLY") && !get_var("DUALBOOT") && !get_var("RESCUECD") && !check_var('FLAVOR', 'Gnome-Live');
```

Comment:
why should that not apply for gnome-live when we have that for the kde live media?